### ISSUE TITLE
Not Working with University of Saskatchewan Canvas page

### ISSUE THREAD

**Issue by GrimmsterZed** [canvasplus/extension#53]
I hope this is the right place to report the bug, but it appears that this plugin does not work with my school's canvas page (University of Saskatchewan). I saw that you are addressing these issues, and I hope this can help!

**Comment by adriancasares** [comment]
Yeah, this will be in the Canvas+ overhaul, but unfortunately it won't be out for a while. Thank you for the concern!
